atlasdb-client/src/integrationInput/java/com/palantir/atlasdb/table/description/generated/SchemaApiTestTable.java

```diff
@@ -871,6 +870,7 @@ public List<String> findConstraintFailuresNoRead(Map<Cell, byte[]> writes,
      * {@link UUID}
      * {@link UnsignedBytes}
      * {@link ValueType}
+     * {@link javax.annotation.processing.Generated}
```

Comment:
Unsure how much you care that this changed...

Reply:
This is fine. I believe these are just generated to avoid warnings in generated code about unused imports as the renderers conditionally generate some code blocks, and some are more naive than others. I think gradle-baseline may also properly configure checkstyle and error-prone to ignore warnings from generated code, so that may no longer be an issue.